### PR TITLE
Storybook Pills documentation page

### PR DESCRIPTION
**Pull request by iandilling-adaptive**

### CODE REVIEW

File: packages/ui-kit-components/src/components/Pill/Pill.tsx [hunk 1]
```diff
@@ -2,25 +2,44 @@ import { Chip } from '@mui/material';
...
 
 export interface PillProps {
   label: string;
-  variant?: any;
+  variant?: PillVariants;
+  color?: PillColors;
   ticker?: boolean;
   disabled?: boolean;
   icon?: JSX.Element;
   rightIcon?: JSX.Element;
```
Comment (wenfeng-xing) [inline packages/ui-kit-components/src/components/Pill/Pill.tsx:20]: Only rightIcon is available?

**Review by wenfeng-xing — APPROVED** [review]

File: packages/ui-kit-components/src/components/Pill/Pill.tsx (hunk 1 above)
Comment (iandilling-adaptive) [reply packages/ui-kit-components/src/components/Pill/Pill.tsx:20]: The `icon` field is for the left icon.

**Comment by iandilling-adaptive** [comment]
Screen Shot 2022-10-28 at 12 15 33 PM
Screen Shot 2022-10-28 at 12 15 50 PM

File: docs/src/design-system/pills/pills-badges-usage.tsx [hunk 2]
```diff
@@ -0,0 +1,175 @@
+import { Box, Container, Toolbar, Typography, useTheme } from '@mui/material';
```
Comment (Honglin-Lu) [inline docs/src/design-system/pills/pills-badges-usage.tsx:1]: Is there any difference between `import {Typography} from '@mui/material'` and `import { Typography } from '@ui-kit-2022/components';`? I tested that they had the same effect on our typography font styles.

Comment (iandilling-adaptive) [reply docs/src/design-system/pills/pills-badges-usage.tsx:1]: Ah, I forgot there was on in the library, too. I took a look, and the Typography component from `@ui-kit-2022/components` is just a reexported version of the one from MaterialUI, so they are the same.

**Comment by Honglin-Lu** [comment]
> Screen Shot 2022-10-28 at 12 15 33 PM
> 
> Screen Shot 2022-10-28 at 12 15 50 PM

I think I get some inspiration from you pills documentation page!

**Review by Honglin-Lu — APPROVED** [review]

**Pull request merged**